Tests/function/tests/Update-VSTeamGitRepositoryDefaultBranch.Tests.ps1

```diff
@@ -0,0 +1,62 @@
+Set-StrictMode -Version Latest
+
+Describe "VSTeamGitRepository" {
+   BeforeAll {
+      . "$PSScriptRoot\_testInitialize.ps1" $PSCommandPath
+      . "$PSScriptRoot\..\..\..\Source\Private\common.ps1"
+
+      Mock Invoke-RestMethod { Open-SampleFile 'Get-VSTeamGitRepository.json' }
+      Mock Invoke-RestMethod { Open-SampleFile 'Get-VSTeamGitRepository-ProjectNamePeopleTracker-NamePeopleTracker.json' } -ParameterFilter {
+         $Uri -like "*00000000-0000-0000-0000-000000000000*" -or $URI -like "*Peopletracker*"
+      }
+      
+      Mock Invoke-RestMethod { throw [System.Net.WebException] } -ParameterFilter {
+         $Uri -like "*00000000-0000-0000-0000-000000000101*" -or
+         $Uri -like "*boom*"
+      }
+   }
+
+   Context 'Update-VSTeamGitRepositoryDefaultBranch' {
+      Context 'Services' {
+         BeforeAll {
+            ## Arrange
+            Mock _getInstance { return 'https://dev.azure.com/Test' }
+         }
+
+         It "by name should update Git repo's default branch" {
+            ## Act
+            Update-VSTeamGitRepositoryDefaultBranch -Name PeopleTracker -projectname PeopleTracker -DefaultBranch 'develop'
+
+            ## Assert
+            Should -Invoke Invoke-RestMethod -ParameterFilter {
+               Write-Debug "Method : $Method"
```

Comment:
Mhh this works? Never thought of that!

Reply:
The debug lines (meant to remove, again sorry -- working on this _at work_ while working on some migration stuff so in a time crunch) or the assertion itself?